bite-size-notes/week-8-e-Bayes Rule-RBE-RBF.md

```diff
@@ -0,0 +1,94 @@
+# Bayes Rule, Recursive Bayesian Estimator, and the Recursive Bayesian Filter
+
+## Bayes's Rule
+
+Bayes's rule, named after Reverend Thomas Bayes, is a fundamental principle in probability theory and statistics. It describes how to update the probability of a hypothesis based on new evidence. Mathematically, Bayes's rule is expressed as:
+$$P(X|Y) = \frac{P(Y|X)P(X)}{P(Y)}$$
+
+Each component of the expression is important and has its own name:
+- $P(X|Y)$ is the *posterior*, which asks "if y is a measurement, then what is x given y?"
+- $P(X)$ is the *prior*, a belief/guess of X before knowing Y
+- $P(Y|X)$ is the *likelihood function*, which is what you expect Y to be given X
+- $P(Y)$ (also written as $\int_X P(Y|X)P(X)dX$) is the normalizing factor.
+
+Yet what happens if we have multiple measurements? We can write our multiple measurement expression as the following:
+$$P(X|Y_1, Y_2) \propto P(Y_2|X,Y_1)P(X|Y_1)$$
+
+Keep in mind, we assume our measurements are independent:
+$$\propto P(Y_2|X)P(X|Y_1)$$
+$$\propto P(Y_2|X)P(Y_1|X)P(X)$$
+
+This applies for any number of measurements $P(X|Y_1,Y_2,\cdots, Y_N)$.
+
+## Recursive Bayes's Estimator
+
+We use the recursive Bayes's estimator to update the estimate of an unknown quantity as new data becomes available through the application of Bayes's Rule. It can be written as the following expression:
+$$P(X|Y_{1:K}) = \frac{P(Y_K|X)P(X|Y_{1:K-1})}{\int_{\tilde{X}} P(Y_K|\tilde{X})P(\tilde{X}|Y_{1:K-1})d\tilde{X}}$$
+
+A tilde is added to the integration term in the denominator in our notation for clarity. As well, note that the prior $P(X|Y_{1:K-1})$ for time step $k$ is simply the posterior from the previous time step $k-1$. Using this, we can then build the Recursive Bayesian Filter, but a few properties and mathematical expressions are worth going over first. The LHS of the RBE is proportional to the following expression:
```

Comment:
To clarify what? Maybe say "to clarify that it is a dummy variable"

Reply:
The tilde is there to clarify that that is the specific term being integrated in the denominator. I'm using the notation that Dr. Leung used when we went over this in class where she added the same thing. I'll try and rephrase it differently to make its use more clear.